app/models/fuel/author.rb

```diff
@@ -11,8 +11,16 @@ def full_name
       [first_name, last_name].compact.join(" ")
     end
 
-    has_attached_file :avatar, :styles => { :medium => Fuel.configuration.avatar_settings[:styles][:medium], :thumb => Fuel.configuration.avatar_settings[:styles][:thumb] }, :default_url => "fuel/default-img.jpg"
+    if Fuel.configuration.aws_bucket
+      has_attached_file :avatar, :styles => { :medium => Fuel.configuration.avatar_settings[:styles][:medium], :thumb => Fuel.configuration.avatar_settings[:styles][:thumb] }, :default_url => "fuel/default-img.jpg", :storage => :s3, :s3_credentials => Proc.new{|a| a.instance.s3_credentials }
```

Comment:
Line is too long. [292/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.<br>Space between { and | missing.